src/mlpack/tests/cf_test.cpp

```diff
@@ -34,7 +34,7 @@
 #include <mlpack/methods/cf/interpolation_policies/regression_interpolation.hpp>
 
 #include <iostream>
-
+#include <math.h>
```

Comment:
There should be no need to include this---it should be already included by `mlpack/prereqs.hpp`.